### ISSUE TITLE
Fix Request: false positives with MissingEqualsAndHashCodeWithGetterSetter

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#528]
Rule ImplementEqualsHashCodeOnValueObjects has Throwable excluded, this rule should do the same.

```
@Getter // <- false positive
@ToString
public class TechnicalException extends ServiceException {
    private final Throwable originalException;

    public TechnicalException(String message) {
        super(message);
        originalException = null;
    }
    public TechnicalException(String message, final Throwable originalException) {
        super(message);
        this.originalException = originalException;
    }
}
```
